src/lib.rs

```diff
@@ -12,13 +12,17 @@ pub struct Configuration {
     /// Keep the originally set panic hook, continuing any normal panic behaviour
     /// and custom panic behaviour set.
     pub keep_original_hook: bool,
+
+    /// Flush this logger after logging the panic.
```

Comment:
```suggestion
    /// When set, flush the given logger after log is called
```

Probably need to think of a few design elements here, ultimately. Giving it some thought, it probably doesn't make sense that we're not allowing specifying the logger for... logging. But I'll split that out into an issue if some things need to be different, this is a good addition nevertheless.

Reply:
Updated the docstring :)